### ISSUE TITLE
Unreproducible results with 6 references

### ISSUE THREAD

**Issue by ranpox** [henryhungle/MTN#6]
Hi there, 
thanks for your great work. I found that you provided the scores with 6 references on this issue #5. Even though I can reproduce the scores with single reference, the scores with 6 references are still unreproducible. 
I simply used the provided code and hyperparameters and dropped the result file into evaluation tools. Then I got following scores:

~~~
./run.sh 1 "vggish i3d_flow" "vggish+i3d_flow"20 9660 0.2
~~~

1 reference:
Bleu_4: 0.129
METEOR: 0.162
ROUGE_L: 0.359
CIDEr: 1.243

6 references:
Bleu_4: 0.344
METEOR: 0.269
ROUGE_L: 0.555
CIDEr: 1.034

I especially found that you didn't manually seed for PyTorch. 
https://github.com/henryhungle/MTN/blob/76491139e03ee4db18348a55c5e6b4136de90735/train.py#L108-L109
I'm not sure if this is the major factor. Could you share your hyperparameters or evaluation method?

**Comment by henryhungle** [comment]
Sorry for the late reply. The results I got was trained on a small version of the model (d=512) but with 16 heads. I also tried with a dropout rate of 0.5, larger warmup steps (e.g. 13k), and more training epochs (max 30 epochs).

**Comment by Leeyouxie** [comment]
> Sorry for the late reply. The results I got was trained on a small version of the model (d=512) but with 16 heads. I also tried with a dropout rate of 0.5, larger warmup steps (e.g. 13k), and more training epochs (max 30 epochs).

Can you show the scores of 6 references with the above configure?

**Comment by henryhungle** [comment]
it was the one I reported in #5

**Comment by ranpox** [comment]
Thank you so much ! @henryhungle
